### PR TITLE
some coding before connectin websocket and configuring the router

### PR DESCRIPTION
**Pull request by ReginaVarybrus**

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -6,20 +6,11 @@ export const GlobalStyles = createGlobalStyle`
 /* !!!Додати стилі після затвердження макету!!! */
```
Comment (DimaShubin) [inline src/GlobalStyles.js]: Let's write comments in English.

File: src/components/Chat/Chat.jsx [hunk 2]
```diff
@@ -1,39 +1,95 @@
...
+
+  const [messageBlockHeight, setMessageBlockHeight] = useState(0);
+  const [value, setValue] = useState('');
+
+  // const fileInput = React.createRef();
+  const textAreaRef = useRef(null);
+
+  const handleChange = e => {
+    setValue(e.target.value);
+    console.log(value);
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: In the main branch should be production-ready version of code. So let's remove everything redundant for prod build.

File: src/components/Chat/Chat.jsx [hunk 3]
```diff
@@ -1,39 +1,95 @@
...
+    const handleResize = () => {
+      updateMessageBlockHeight();
+    };
+  
+    window.addEventListener('resize', handleResize);
+  
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: Line 37-59. Why do you need this? Try to use useEffect as less as possible.

File: src/components/Chat/Chat.jsx [hunk 4]
```diff
@@ -1,39 +1,95 @@
...
 
   return (
     <Wrapper>
       <Header>
         <HeaderContent>
-          <h5>Country Name</h5>
-          <p>members</p>
+          {!countryName ? <h5>Country Name</h5> : <h5>{countryName}</h5>}
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]:
```suggestion
          {<h5>{countryName || "Country Name"}</h5>}
```

File: src/components/Chat/Chat.jsx [hunk 5]
```diff
@@ -1,39 +1,95 @@
...
+          {participants != null ? (
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx:66]:
```suggestion
          {participant.length ? (
```

Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]:
```suggestion
          {<p>{participants || 0} members</p> }
```

File: src/components/Chat/Chat.jsx [hunk 6]
```diff
@@ -1,39 +1,95 @@
...
+            <p>{participants} members</p>
+          ) : (
+            <p>0 members</p>
+          )}
         </HeaderContent>
       </Header>
-      <MessageBlock></MessageBlock>
+      <MessageBlock style={{ height: messageBlockHeight + 'px' }}></MessageBlock>
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: I believe we'll get red of this.

File: src/components/Chat/Chat.jsx [hunk 7]
```diff
@@ -1,39 +1,95 @@
...
       <MessageBarWrapper>
         <MessageBar>
-          <Button>
+          {/* <ButtonAddFile type="file" ref={fileInput}> */}
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: Don't leave comments like this. Only comments aimed to explain something.

File: src/components/Chat/Chat.jsx [hunk 8]
```diff
@@ -1,39 +1,95 @@
...
+          <ButtonAddFile>
             <Icons name="attach-file" fill="var(--color-grey-9)" size="24" />
-          </Button>
-          <Input type="text" placeholder="Type here" />
-          <Button>
-          <Icons name="send" fill="var(--color-grey-9)" size="24" />
-          </Button>
+          </ButtonAddFile>
+          <Input
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: I've noticed you used MUI library across the project. Why didn't you use MUI input component here?

File: src/components/Chat/Chat.jsx [hunk 9]
```diff
@@ -1,39 +1,95 @@
...
+            type="text"
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: Validation should be added as well.

File: src/components/Chat/Chat.jsx [hunk 10]
```diff
@@ -1,39 +1,95 @@
...
+            placeholder="Type here"
+            value={value}
+            onChange={handleChange}
+            ref={textAreaRef}
+            maxLength="1000"
+            rows="1"
+            style={{height: '48px'}}
+          />
+          <ButtonSendMessage style={{ border: isInputNotEmpty ? '1px solid var(--color-brand-blue)' : '1px solid var(--color-grey-6)' }}>
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: Styles should be in appropriate file = './ChatStyled';

File: src/components/Chat/ChatStyled.js [hunk 11]
```diff
@@ -32,43 +33,89 @@ export const HeaderContent = styled.div`
...
+  transition: height 0.3s ease;
 `;
 
 export const MessageBarWrapper = styled.footer`
   width: 100%;
   min-height: 112px;
+  max-height: 444px;
   background: var(--white-color);
-  border-top: 1px solid var(--color-grey-6);
+  border-top: 1px solid var(--color-grey-6);width: 100%;
```
Comment (DimaShubin) [inline src/components/Chat/ChatStyled.js]: Formating.

File: src/components/ChatMap/ChatMap.jsx [hunk 12]
```diff
@@ -1,47 +1,50 @@
...
+  getUserId,
+  getPersistedToken,
+} from 'redux-store/AuthOperations/selectors';
+import { CountryName, ShowCountry, MainMapBlock, MapWrapper } from './ChatMapStyled';
 // import { TileLayer } from 'react-leaflet/TileLayer';
 // import { useMap } from 'react-leaflet/hooks';
 // import { TileLayer, useMap } from 'react-leaflet';
 
-export default function ChatMap() {
+function ChatMap({closeMap}) {
```
Comment (DimaShubin) [inline src/components/ChatMap/ChatMap.jsx]: Let's create components using arrow function approach. 
```suggestion
const ChatMap = ({ closeMap }) => {
```

File: src/components/ChatMap/ChatMap.jsx [hunk 13]
```diff
@@ -59,39 +62,59 @@ export default function ChatMap() {
           name: e.target.feature.properties.ADMIN,
           flagCode: e.target.feature.properties.code,
         };
-        dispatch(sendDataCountryToBackend(userId, data));
-        console.log('data to send', userId, data);
+        setSelectedCountry(data.name);
```
Comment (DimaShubin) [inline src/components/ChatMap/ChatMap.jsx:65]: Total mess. Defiantly should be refactored.

File: src/components/ChatMap/dist/ChatMapStyled.dev.js [hunk 14]
```diff
@@ -0,0 +1,68 @@
+"use strict";
```
Comment (DimaShubin) [inline src/components/ChatMap/dist/ChatMapStyled.dev.js]: What is that file for?

File: src/components/DMsList/DMsListStyled.js [hunk 15]
```diff
@@ -7,5 +7,60 @@ export const Wrapper = styled.div`
...
+export const MessageDay = styled.div`
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-end;
+  width: 40px;
+  height: 50px;
+  & > p {
+    color: var(--color-grey-9);
+  }
 `;
```
Comment (DimaShubin) [inline src/components/DMsList/DMsListStyled.js]: Empty line.

File: src/components/DMsList/dist/DMsListStyled.dev.js [hunk 16]
```diff
@@ -0,0 +1,110 @@
+"use strict";
```
Comment (DimaShubin) [inline src/components/DMsList/dist/DMsListStyled.dev.js]: Wtf?

File: src/components/RoomsList/dist/RoomsListStyled.dev.js [hunk 17]
```diff
@@ -0,0 +1,82 @@
+"use strict";
```
Comment (DimaShubin) [inline src/components/RoomsList/dist/RoomsListStyled.dev.js]: Are you kidding me?)

File: src/components/Chat/Chat.jsx (hunk 2 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: I need explain it.

File: src/components/Chat/Chat.jsx (hunk 3 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: I need that this textarea for a message to increase when a text of the message bigger than one string so I founded only  this solution for this task. I need your advice what another way i can solution this.

File: src/components/Chat/Chat.jsx (hunk 6 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: It`s also about increasing textarea for message

File: src/components/Chat/Chat.jsx (hunk 9 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: What type of validation?

File: src/components/ChatMap/ChatMap.jsx (hunk 13 above)
Comment (ReginaVarybrus) [reply src/components/ChatMap/ChatMap.jsx:65]: Ok, I need your help with it

File: src/components/ChatMap/dist/ChatMapStyled.dev.js (hunk 14 above)
Comment (ReginaVarybrus) [reply src/components/ChatMap/dist/ChatMapStyled.dev.js]: I don't know

File: src/components/DMsList/DMsListStyled.js (hunk 15 above)
Comment (ReginaVarybrus) [reply src/components/DMsList/DMsListStyled.js]: What line is empty?

File: src/components/RoomsList/dist/RoomsListStyled.dev.js (hunk 17 above)
Comment (ReginaVarybrus) [reply src/components/RoomsList/dist/RoomsListStyled.dev.js]: Yes)

File: src/components/Chat/Chat.jsx (hunk 2 above)
Comment (DimaShubin) [reply src/components/Chat/Chat.jsx]: https://medium.com/@himanshuganglani/clean-code-how-and-when-to-comment-80a83272a9c8#:~:text=Avoid%20commented%2Dout%20code%20in,the%20courage%20to%20delete%20it.

File: src/components/Chat/Chat.jsx (hunk 9 above)
Comment (DimaShubin) [reply src/components/Chat/Chat.jsx]: Yup + formik for input field validation. https://formik.org/docs/guides/validation

File: public/index.html [hunk 18]
```diff
@@ -40,4 +40,4 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
-</html>
+</html>
```
Comment (DimaShubin) [inline public/index.html]: Empty line.

File: src/components/Chat/Chat.jsx [hunk 19]
```diff
@@ -17,81 +18,49 @@ import {
...
-            <p>{participants} members</p>
-          ) : (
-            <p>0 members</p>
-          )}
+          {<h5>{countryName || 'Country Name'}</h5>}
+          {<p>{participants || 0} members</p>}
         </HeaderContent>
       </Header>
-      <MessageBlock style={{ height: messageBlockHeight + 'px' }}></MessageBlock>
+      <MessageBlock></MessageBlock>
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]: `<MessageBlock />`

File: src/components/Chat/Chat.jsx [hunk 20]
```diff
@@ -1,41 +1,66 @@
...
+          </ButtonAttachFile>
+          <TextareaAutosize
+            aria-label="empty textarea"
+            placeholder="Type here"
+            value={value}
+            onChange={handleChange}
+            ref={textAreaRef}
+            maxLength="1000"
+          />
+          <ButtonSendMessage border={isInputNotEmpty}>
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]:
```suggestion
          <ButtonSendMessage isInputNotEmpty>
```

File: src/components/Chat/Chat.jsx [hunk 21]
```diff
@@ -1,41 +1,66 @@
...
+const Chat = () => {
+  const countryName = useSelector(getCountryName);
+  const participants = useSelector(getNumberOfParticipants);
+  const [value, setValue] = useState('');
+
+  const textAreaRef = useRef(null);
 
-export default function Chat() {
-  // const countryName = useSelector(getCountryName);
+  const isInputNotEmpty = value.trim() !== '';
```
Comment (DimaShubin) [inline src/components/Chat/Chat.jsx]:
```suggestion
  const isInputNotEmpty = Boolean(value?.trim().length);
```

File: src/components/ChatMap/ChatMapStyled.js [hunk 22]
```diff
@@ -1,8 +1,28 @@
 import styled from 'styled-components';
```
Comment (DimaShubin) [inline src/components/ChatMap/ChatMapStyled.js:1]: Wrong formatting

File: src/components/Layout/Layout.jsx [hunk 23]
```diff
@@ -1,3 +1,4 @@
+import { Suspense, useState } from 'react';
```
Comment (DimaShubin) [inline src/components/Layout/Layout.jsx]: Redundant code.

File: src/components/Layout/LayoutStyled.js [hunk 24]
```diff
@@ -1,24 +1,25 @@
...
-//   }
-// `;
-
 export const LayoutStyled = styled.div`
   display: grid;
   grid-template-columns: 120px 1fr;
   background-color: var(--outlet-background-color);
 `;
+
+export const LogoBlock = styled.div`
```
Comment (DimaShubin) [inline src/components/Layout/LayoutStyled.js]: I don't see any places you use these two components. Do we really need it?

File: src/components/Layout/LayoutStyled.js [hunk 25]
```diff
@@ -1,24 +1,25 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  & > p {
+    color: var(--color-dark);
+  }
+`;
+
+export const Logo = styled.img`
```
Comment (DimaShubin) [inline src/components/Layout/LayoutStyled.js]: Same as above

File: src/components/RoomsList/RoomsList.jsx [hunk 26]
```diff
@@ -1,19 +1,33 @@
...
-      </Wrapper>
-    );
-  }
-  
\ No newline at end of file
+  const handleClick = () => {};
+
+  return (
+    <Wrapper>
+      {rooms.length === 0 && (
```
Comment (DimaShubin) [inline src/components/RoomsList/RoomsList.jsx]:
```suggestion
      {rooms?.length && (
```

File: src/components/RoomsList/RoomsList.jsx [hunk 27]
```diff
@@ -1,19 +1,33 @@
...
+        <Text>
+          There are no rooms in the list.
+          <br /> Find chat of a country and it will be shown here
+        </Text>
+      )}
+      <ListItems>
+        <SimpleBar style={{ maxHeight: 570 }}>
```
Comment (DimaShubin) [inline src/components/RoomsList/RoomsList.jsx]: Let's avoid using inline styles

File: src/components/RoomsList/RoomsListStyled.js [hunk 28]
```diff
@@ -7,5 +7,35 @@ export const Wrapper = styled.div`
...
+  &:hover {
+    background: var(--color-blue-1);
+  }
+`;
+
+export const Flag = styled.img`
+  width: 32px;
+  height: 24px;
+  padding-right: 12px;
 `;
```
Comment (DimaShubin) [inline src/components/RoomsList/RoomsListStyled.js]: Missed empty line.

File: src/components/SearchInput/SearchInput.jsx [hunk 29]
```diff
@@ -96,7 +85,7 @@ export default function SearchInput() {
       {showItem && (
         <ListWrapper>
           <ListItems>
-            <SimpleBar style={{ maxHeight: 570 }}>
+            <SimpleBar style={{ maxHeight: 570 }} autoHide={false}>
```
Comment (DimaShubin) [inline src/components/SearchInput/SearchInput.jsx]: 1. Inline styles. 
2. Just remove autoHide={false}

File: src/components/SideBar/SideBar.jsx [hunk 30]
```diff
@@ -1,101 +1,87 @@
...
   const userName = useSelector(getUserName);
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const location = useLocation();
+  const currentPage = location.pathname;
 
   const handleProfileOpen = () => {
-    navigate('/app/account');
-    console.log('clicked profile');
+    navigate('account');
```
Comment (DimaShubin) [inline src/components/SideBar/SideBar.jsx]: Use routesPath constant for navigating between routes.

File: src/components/SideBar/SideBar.jsx [hunk 31]
```diff
@@ -1,101 +1,87 @@
...
   };
 
-  const handleLogOut = (event, values) => {
-    dispatch(logOut(values));
-    navigate('/');
-    console.log('clicked logout', event);
+  const handleLogOut = async (event, values) => {
+    try {
+      await dispatch(logOut(values));
+      navigate('/');
```
Comment (DimaShubin) [inline src/components/SideBar/SideBar.jsx]: Use routesPath constant for navigating between routes.

File: src/components/SideBar/SideBar.jsx [hunk 32]
```diff
@@ -1,101 +1,87 @@
...
+      console.log('Logout successful', event);
```
Comment (DimaShubin) [inline src/components/SideBar/SideBar.jsx]: redundant

File: src/components/SideBar/SideBar.jsx [hunk 33]
```diff
@@ -1,101 +1,87 @@
...
+    } catch (error) {
+      console.error('Error during logout', error);
```
Comment (DimaShubin) [inline src/components/SideBar/SideBar.jsx]: redundant

File: src/components/SideBar/SideBar.jsx [hunk 34]
```diff
@@ -1,101 +1,87 @@
...
-          onClick={handleProfileOpen}
-          src=""
-          alt="Profile Avatar"
-        >
-          <Icons name='photo-icon' color='#222222' size='24' />
-        </ProfileBoxAvatar>
-        <Text>Anton</Text> */}
         <SideBarItem
           onClick={handleProfileOpen}
+          isActive={currentPage === '/app/account'}
```
Comment (DimaShubin) [inline src/components/SideBar/SideBar.jsx]: routesConfig constant for routes names.

File: src/components/SideBarItem/SideBarItem.jsx [hunk 35]
```diff
@@ -3,26 +3,23 @@ import PropTypes from 'prop-types';
...
+  children
+}) => {
   return (
     <SideBarButton
       onClick={() => onClick()}
-      style={{
-        color: isActive ? 'var(--color-brand-blue)' : 'var(--color-grey-9)',
-        fontWeight: isActive ? '700' : '400',
-      }}
+      color={isActive}
```
Comment (DimaShubin) [inline src/components/SideBarItem/SideBarItem.jsx]: Put the isActive prop down and use it to manipulate with color and fontWeight properties.

File: src/css/ChatMap.css [hunk 36]
```diff
@@ -13,9 +6,12 @@
...
 
 .leaflet-control-zoom {
-    position: absolute;
-    top: 696px;
-    left: 533px;
+    /* position: absolute;
+    top: 655px;
+    left: 50%; */
+    /* bottom: 50px; */
+    /* transform: translate(-50%, 0); */
```
Comment (DimaShubin) [inline src/css/ChatMap.css:13]: comments?

File: src/public/icons/icon-about.svg
Comment (DimaShubin) [inline src/public/icons/icon-about.svg:1]: Why did you move all svg files to the public folder?

Comment (DimaShubin) [reply src/public/icons/icon-about.svg:1]: Ok seems like you did it coincidentally but you created folder public in src/ and moved all svg icons there?

File: src/redux-store/AuthOperations/selectors.js [hunk 37]
```diff
@@ -8,4 +8,6 @@ export const getUserRefresh = state => state.auth.isRefresh;
 export const getUser = state => state.auth;
 export const getPersistedToken = state => state.auth.token;
 
-export const getCountryName = state => state.auth.countryDto.name;
+export const getCountryName = state => state.auth.name;
+export const getNumberOfParticipants = state => state.auth.participants;
+export const countryRooms = state => state.room.countryRooms;
```
Comment (DimaShubin) [inline src/redux-store/AuthOperations/selectors.js]: empty line should be in the end of the file.

File: src/redux-store/rootReducer.js [hunk 38]
```diff
@@ -0,0 +1,10 @@
+// import { combineReducers } from '@reduxjs/toolkit';
+// import { authSlice } from './slices/authSlice';
+// import roomReducer from './slices/roomSlise';
+
+// const rootReducer = combineReducers({
+//   auth: authSlice.reducer,
+//   room: roomReducer,
+// });
```
Comment (DimaShubin) [inline src/redux-store/rootReducer.js]: ????

File: src/redux-store/rootReducer.js [hunk 39]
```diff
@@ -0,0 +1,10 @@
...
+
+// export default rootReducer;
```
Comment (DimaShubin) [inline src/redux-store/rootReducer.js]: empty line.

File: src/redux-store/slices/roomSlise.js [hunk 40]
```diff
@@ -0,0 +1,17 @@
...
+//   reducers: {
+//     addCountryRoom: (state, action) => {
+//       state.countryRooms = [...new Set([...state.countryRooms, action.payload])];
+//     },
+//   },
+// });
+
+// export const { addCountryRoom } = roomSlice.actions;
+
+// export default roomSlice.reducer;
```
Comment (DimaShubin) [inline src/redux-store/slices/roomSlise.js]: delete this file.

File: src/redux-store/store.js [hunk 41]
```diff
@@ -1,4 +1,5 @@
 import { configureStore } from '@reduxjs/toolkit';
+// import rootReducer from './rootReducer';
 import {
```
Comment (DimaShubin) [inline src/redux-store/store.js]: ???

**Review by DimaShubin — CHANGES_REQUESTED** [review]

File: src/components/Chat/Chat.jsx (hunk 19 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: Done

File: src/components/Chat/Chat.jsx (hunk 21 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: Done

File: src/redux-store/AuthOperations/selectors.js (hunk 37 above)
Comment (ReginaVarybrus) [reply src/redux-store/AuthOperations/selectors.js]: Done

File: src/redux-store/slices/roomSlise.js (hunk 40 above)
Comment (ReginaVarybrus) [reply src/redux-store/slices/roomSlise.js]: Done

File: src/redux-store/store.js (hunk 41 above)
Comment (ReginaVarybrus) [reply src/redux-store/store.js]: Done

File: src/redux-store/rootReducer.js (hunk 39 above)
Comment (ReginaVarybrus) [reply src/redux-store/rootReducer.js]: Done

File: src/redux-store/rootReducer.js (hunk 38 above)
Comment (ReginaVarybrus) [reply src/redux-store/rootReducer.js]: I deleted this file

File: src/components/Layout/Layout.jsx (hunk 23 above)
Comment (ReginaVarybrus) [reply src/components/Layout/Layout.jsx]: Done

File: src/components/Layout/LayoutStyled.js (hunk 24 above)
Comment (ReginaVarybrus) [reply src/components/Layout/LayoutStyled.js]: Yes it will be first loading of Chat if the user hasn't any opened chat room yet. But I can delete it now and add it later

File: src/components/RoomsList/RoomsList.jsx (hunk 26 above)
Comment (ReginaVarybrus) [reply src/components/RoomsList/RoomsList.jsx]: Done

File: src/components/RoomsList/RoomsList.jsx (hunk 27 above)
Comment (ReginaVarybrus) [reply src/components/RoomsList/RoomsList.jsx]: Moved style to the StyledComponent

File: src/components/RoomsList/RoomsListStyled.js (hunk 28 above)
Comment (ReginaVarybrus) [reply src/components/RoomsList/RoomsListStyled.js]: Done

File: src/components/SearchInput/SearchInput.jsx (hunk 29 above)
Comment (ReginaVarybrus) [reply src/components/SearchInput/SearchInput.jsx]: Done

File: src/components/SideBar/SideBar.jsx (hunk 30 above)
Comment (ReginaVarybrus) [reply src/components/SideBar/SideBar.jsx]: Done

File: src/components/SideBar/SideBar.jsx (hunk 31 above)
Comment (ReginaVarybrus) [reply src/components/SideBar/SideBar.jsx]: Done

File: src/components/SideBar/SideBar.jsx (hunk 32 above)
Comment (ReginaVarybrus) [reply src/components/SideBar/SideBar.jsx]: Done

File: src/components/SideBar/SideBar.jsx (hunk 33 above)
Comment (ReginaVarybrus) [reply src/components/SideBar/SideBar.jsx]: Done

File: src/components/SideBar/SideBar.jsx (hunk 34 above)
Comment (ReginaVarybrus) [reply src/components/SideBar/SideBar.jsx]: Done

File: src/css/ChatMap.css (hunk 36 above)
Comment (ReginaVarybrus) [reply src/css/ChatMap.css:13]: Can we leave this for task TAL-55 Search by map and I'll move this styles to ChatMapStyled and clear file?

File: src/public/icons/icon-about.svg
Comment (ReginaVarybrus) [reply src/public/icons/icon-about.svg:1]: Done

File: src/components/SideBarItem/SideBarItem.jsx (hunk 35 above)
Comment (ReginaVarybrus) [reply src/components/SideBarItem/SideBarItem.jsx]: Already moved it in SideBarItemStyled.js

File: src/components/Chat/Chat.jsx (hunk 20 above)
Comment (ReginaVarybrus) [reply src/components/Chat/Chat.jsx]: Done

File: src/components/ChatMap/ChatMapStyled.js (hunk 22 above)
Comment (ReginaVarybrus) [reply src/components/ChatMap/ChatMapStyled.js:1]: Can we leave this for task TAL-55 Search by map and I'll move this styles to ChatMapStyled and clear file?

**Review by DimaShubin — APPROVED** [review]
